lib/features/manage_account/presentation/profiles/identities/identities_controller.dart

```diff
@@ -195,19 +223,49 @@ class IdentitiesController extends BaseController {
       DeleteIdentityDialogBuilder(
         responsiveUtils: responsiveUtils,
         imagePaths: imagePaths,
-        onDeleteIdentityAction: () => _deleteIdentityAction(identity),
+        onDeleteIdentityAction: () => _dereferencePublicAssets(identity),
       ),
       barrierColor: AppColor.colorDefaultCupertinoActionSheet,
     );
   }
 
-  void _deleteIdentityAction(Identity identity) {
+  Future<void> _deleteIdentityAction(IdentityId? identityId) async {
+    final session = accountDashBoardController.sessionCurrent;
+    final accountId = accountDashBoardController.accountId.value;
+    if (accountId != null && session != null && identityId != null) {
+      consumeState(_deleteIdentityInteractor.execute(session, accountId, identityId));
+    }
+  }
+
+  Future<void> _dereferencePublicAssets(
+    Identity identity
+  ) async {
     popBack();
 
     final session = accountDashBoardController.sessionCurrent;
     final accountId = accountDashBoardController.accountId.value;
-    if (accountId != null && session != null && identity.id != null) {
-      consumeState(_deleteIdentityInteractor.execute(session, accountId, identity.id!));
+    final identityId = identity.id;
+
+    final publicAssetIds = identity
+      .signatureAsString
+      .publicAssetIdsFromHtmlContent;
+    final removeIdentityFromPublicAssetsInteractor 
+      = getBinding<RemoveIdentityFromPublicAssetsInteractor>(
+        tag: BindingTag.cleanUpPublicAssetsInteractorBindingsTag);
```

Comment:
ident

Reply:
I don't think there is any problem with indent on that part

Reply:
As I see above `tag` are on par with `getBinding `

Reply:
`tag` is indent with equal sign `=`. Please check again.

Reply:
Nevermind. Updated